### PR TITLE
ENG-2175 Pages management tests added and other improvements

### PR DESCRIPTION
**Pull request by alepintus**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **alepintus**

If they are not already assigned, you can assign the PR to them by writing `/assign @alepintus` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: test/cypress/support/command/command.js
Comment (gidesan) [inline test/cypress/support/command/command.js:27]: seems that it's not using ARIA label

File: test/cypress/support/command/page-command.js
Comment (gidesan) [inline test/cypress/support/command/page-command.js:127]: is the `action` suffix really needed? do we need to know the page was published by clicking on the kebab action or elsewhere?

Comment (alepintus) [reply test/cypress/support/command/page-command.js:127]: Yes we have another publish page command in page designer (publishPageClick) and I prefer to differ this from the other (action refers to the actions menu)

File: test/cypress/support/command/command.js
Comment (alepintus) [reply test/cypress/support/command/command.js:27]: nope, this is the only way I found to access properly to the options without changing the typeahead component

Comment (gidesan) [reply test/cypress/support/command/command.js:27]: then please amend ```*  Get Typeahead Option by aria label
 *  @param value - The value of the aria label```

Comment (alepintus) [reply test/cypress/support/command/command.js:27]: Done

File: test/cypress/support/command/command.js
Comment (gidesan) [inline test/cypress/support/command/command.js:27]: then `getTypeaheadOptionByValue` is probably clearer

**Review by gidesan — CHANGES_REQUESTED** [review]

File: test/cypress/support/command/command.js
Comment (alepintus) [reply test/cypress/support/command/command.js]: Done

**Review by gidesan — CHANGES_REQUESTED** [review]
Almost done :)

File: test/cypress/support/command/page-management-command.js
Comment (alepintus) [reply test/cypress/support/command/page-management-command.js]: Commands changed as suggested

File: test/cypress/support/command/page-management-command.js
Comment (alepintus) [reply test/cypress/support/command/page-management-command.js]: Commands changed as suggested

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/1052#pullrequestreview-640788122" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
